libs/ledgerjs/packages/cryptoassets/README.md

```diff
@@ -155,14 +143,6 @@ Returns **(CryptoCurrency | null | [undefined](https://developer.mozilla.org/doc
 
 Returns **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** 
 
-### getCryptoCurrencyById
-
-#### Parameters
-
-*   `id` **any** 
-
-Returns **CryptoCurrency** 
-
```

Comment:
Doesn't seem normal that those functions are being removed from the doc, especially in this PR.

Reply:
https://github.com/LedgerHQ/ledger-live/pull/1713#discussion_r1015257179